layers/+completion/ivy/funcs.el

```diff
@@ -169,10 +169,10 @@ that directory."
                                   spacemacs--counsel-search-max-path-length)
                                (length default-directory))))))
     (cond ((string= tool "ag")
-           (counsel-ag initial-input initial-directory nil
+           (counsel-ag initial-input default-directory nil
                        (format "ag from [%s]: " display-directory)))
           ((string= tool "rg")
-           (counsel-rg initial-input initial-directory nil
+           (counsel-rg initial-input default-directory nil
```

Comment:
I think this is right: `default-directory` is the one that user selects

Reply:
@duianto Any plan to get this merged :) I'm a bit anxious about this bug I introduced

Reply:
Since the search fix was applied to what seems to be an unrelated PR. I asked the maintainers if they had any suggestions for how this should be handled. But I haven't heard back. They are busy.

Do you know anything about TRAMP and could you test if that commit works as expected?

Reply:
My bad about the mess with the commit on unrelated PR. I can create separate branches with proper PR if that'd help.

Regarding TRAMP, I use it for work everyday, and even for accessing my home router (it has a `busybox` `ash` shell). I also use `lsp` with `ccls` to code on C and C++ remotely. When I started using Spacemacs I realized that `SPC f E` was broken for my use case (mostly for editing root config files on the company's server), so I fixed it out of necessity and convenience.

Anyways, I can say that this was only tested with `ssh`, `scp`, and `rsync` protocols, but the logic behind this fix should work for every properly configured TRAMP protocol. Worst case scenario a `Protocol not supported for multi-hopping` message will appear in the minibuffer.